Sources/BraveWallet/Crypto/Onboarding/OnBoardingCompletedView.swift

```diff
@@ -22,38 +22,43 @@ struct OnboardingCompletedView: View {
         keyringStore: keyringStore
       ) { isBiometricCompleted = true }
     } else {
-      ScrollView {
-        VStack {
-          Image("wallet-onboarding-complete", bundle: .module)
-          Text(Strings.Wallet.onboardingCompletedTitle)
-            .font(.title)
-            .foregroundColor(.primary)
-            .fixedSize(horizontal: false, vertical: true)
-          Text(Strings.Wallet.onboardingCompletedSubTitle)
-            .font(.subheadline)
-            .foregroundColor(.secondary)
-            .fixedSize(horizontal: false, vertical: true)
-            .padding(.top, 4)
-          Button {
-            keyringStore.markOnboardingCompleted()
-          } label: {
-            Text(Strings.Wallet.onboardingCompletedButtonTitle)
-              .frame(maxWidth: .infinity)
+      GeometryReader { geometry in
```

Comment:
PR description only mentions tap gesture, what are the other changes for? I don't think the `GeometryReader` is needed?

Reply:
This removes the hardcoded top padding on the VStack `.padding(.top, 124)` to force it fit the entire screen in order to make its subviews stay centred vertically. I initially thought the scrollView was doing some weird stuff that could cause this problem. but its not. left it thinking it's still a good improvement by removing hardcoded top padding.